cabal-install/Distribution/Client/Security/HTTP.hs

```diff
@@ -0,0 +1,307 @@
+{-# LANGUAGE CPP #-}
```

Comment:
This module provides functionality similar to the existing `HttpUtils` module. How difficult would be to merge the two? We can of course do this at a later stage, this is not a requirement for merging.